SourceX/controls/touch.cpp

```diff
@@ -2,6 +2,12 @@
 #include "miniwin/ddraw.h"
 #include "touch.h"
 #include <math.h>
+
+// TODO: How to find display size for each platform programmatically?
```

Comment:
We can use `SDL_GetCurrentDisplayMode`

Reply:
I think the hard coded 1920 should become sdl_displaywidth. 
The hard coded 1080 should become sdl_displayheight
The hard coded 1440 should become (sdl_displayheight*4)/3
That way it will finally be system independent.

Reply:
Maybe this could be done in touch_init?

Reply:
Also be aware that the game may run in other resolutions so you should use SCREEN_WIDTH / SCREEB_HEIGHT instead of 4/3

Reply:
Well I tested that on Switch and it doesn’t work. I think you are drawing the Diablo screen with black borders, but your logical coords ignore the borders.

Reply:
Yes

Reply:
Fixed in latest force-push